package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "sourcecred",
-  "version": "0.7.0-beta-1",
+  "version": "0.7.0-beta-3",
```

should also add in description, homepage, and repository so it shows up on the NPM page:
```suggestion
  "version": "0.7.0-beta-3",
  "description": "A tool for communities to measure and reward value creation",
  "homepage": "https://sourcecred.io",
  "repository": "github:sourcecred/sourcecred",
```

---

Done, thanks. Did change the description, in the (near) future it won't just be the binary but also the API.